src/less/alertify.less

```diff
@@ -825,6 +825,9 @@
             max-height: 100%;
             padding: 15px;
             margin-top: 10px;
+            &.ajs-has-close {
```

Comment:
Don't add `.ajs-` prefix inside the less files, it will be automatically added to the output css during the build process.